tox.ini

```diff
@@ -27,7 +27,7 @@ commands = pytest --durations=3 {posargs: -p no:warnings tests}
 # Bottom of file has Flake8 settings
 commands = flake8 {posargs:diablo config consoler.py application.py scripts tests xena}
 deps =
-    flake8>=5.0.4
+    flake8==5.0.4
```

Comment:
@johncrossman just double checking, is this change supposed to be included in the version bump?

Reply:
@lyttam - Oddly, it does belong.  The latest version of flake8 ain't compatible w/ our Python version so I had to lock it down in order to get green Travis.